RecoLocalTracker/SiPixelRecHits/interface/pixelCPEforDevice.h

```diff
@@ -0,0 +1,434 @@
+#ifndef RecoLocalTracker_SiPixelRecHits_interface_pixelCPEforDevice_h
+#define RecoLocalTracker_SiPixelRecHits_interface_pixelCPEforDevice_h
+
+#include <algorithm>
+#include <cassert>
+#include <cmath>
+#include <cstdint>
+#include <iterator>
+
+#include "DataFormats/SiPixelClusterSoA/interface/ClusteringConstants.h"
+#include "Geometry/CommonTopologies/interface/SimplePixelTopology.h"
+#include "DataFormats/TrackingRecHitSoA/interface/SiPixelHitStatus.h"
+#include "DataFormats/SiPixelClusterSoA/interface/ClusteringConstants.h"
+#include "DataFormats/GeometrySurface/interface/SOARotation.h"
+#include "Geometry/CommonTopologies/interface/SimplePixelTopology.h"
+
+// Nesting this namespace to prevent conflicts with pixelCPEForDevice.h
```

Comment:
What does the "Nesting" refer to?

Reply:
I think this was copied over from the CUDA version.
I've moved these constants into the `pixelCPEforDevice` namespace.